src/Cms/File.php

```diff
@@ -35,6 +35,9 @@ class File extends ModelWithContent
 	use FileActions;
 	use FileModifications;
 	use HasMethods;
+	/**
+	 * @use HasSiblings<\Kirby\Cms\Files>
```

Comment:
Shouldn't those also be fully-qualified:

```suggestion
	 * @use \Kirby\Cms\HasSiblings<\Kirby\Cms\Files>
```